molgenis-data-import/src/main/resources/l10n/data-import_nl.properties

```diff
@@ -3,4 +3,24 @@ EXP01=Er is een fout opgetreden bij het downloaden van uw selectie.
 EXP02=Het download verzoek moet minimaal 1 entiteitsoort of package bevatten.
 EXP03={0, label} ''{1}'' kan niet worden gedownload, de identifier start niet met de (bovenliggende)mapnaam.
 emx_export_progress_message=Bezig met downloaden van ''{0}''
-emx_export_metadata_message=Klaar met downloaden van map metadata
\ No newline at end of file
+emx_export_metadata_message=Klaar met downloaden van map metadata
+IMP01=Niet ondersteunde attribuut metadata: ''{0}'', bedoelde u misschien ''{1}''? (werkblad: ''{2}'')
+IMP02=Dubbele attribuut name ''{0}'' for entiteit type ''{1}''. (werkblad: ''{2}'', rij {3})
+IMP03=Inladen gefaald: {0} (werkblad: ''{1}'', rij {2})
+IMP04=Map naam moet beginnen met de naam van de bovenliggende map. (map: {0}, bovenliggende map: {1} werkblad: ''{2}'', rij {3})
+IMP05=partOfattribuut ''{0}'' van attribuut ''{1}'' van entiteit ''{2}'' moet refereren naar een attribuut van type 'compound'. (werkblad: ''{3}'', rij {4})
```

Comment:
Replace 'compound' with variable

Reply:
because? this only ever applies to compounds